src/v/kafka/server/handlers/alter_client_quotas.cc

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2024 Redpanda Data, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+#include "kafka/server/handlers/alter_client_quotas.h"
+
+#include "kafka/protocol/schemata/alter_client_quotas_request.h"
+#include "kafka/protocol/schemata/alter_client_quotas_response.h"
+
+namespace kafka {
+
+void make_error_response(
```

Comment:
probably want to make this file scope. it's a pretty generic name too be exporting